docs/rosa_create_machinepool.md

```diff
@@ -19,6 +19,9 @@ rosa create machinepool [flags]
   # Add a machine pool mp-1 with 3 replicas of m5.xlarge to a cluster
   rosa create machinepool --cluster=mycluster --name=mp-1 --replicas=3 --instance-type=m5.xlarge
 
+  # Add a machine pool mp-1 with autoscaling enabled and 3 to 6 replicas of m5.xlarge to a cluster
+  rosa create machinepool --cluster=mycluster --name=mp-1 --enable-autoscaling --min-replicas=3 --max-replicas=6 --instance-type=m5.xlarge
```

Comment:
All these docs are auto-generated (`make docs`). You should put this inside the `Example` under each corresponding `&cobra.Command`

Reply:
fixed